__main__.py

```diff
@@ -60,11 +73,30 @@ def main():
         load_balancer.add_server(server)
         servers.append(server)
 
-    env.run(until=SIM_DURATION_SECONDS)
+    before_count = NUMBER_OF_SERVERS * [0]
+    before_time = NUMBER_OF_SERVERS * [0]
+    gc_count = list()  # how much gc executions was runned per second.
```

Comment:
# Number of GC executions per second.

Reply:
I'll fix it on the current branch that I'm work on. Thanks!